views.py

```diff
@@ -373,6 +373,24 @@ def peak_json():
                      'precursor_mz': spectrum.precursor_mz}
     return flask.jsonify(spectrum_dict)
 
+@app.route('/api/proxi/v0.1/spectra')
+def peak_proxi_json():
+    spectrum, _ = parsing.parse_usi(flask.request.args.get('usi'))
+    
+    spectrum_dict = {'intensities': [str(intensity) for intensity in spectrum.intensity],
+                     'mzs' : [str(mz) for mz in spectrum.mz],
```

Comment:
If you want to be super fancy: `list(map(str, spectrum.mz))`. Same for the intensities above.